vcsclient/bitbucketcloud.go

```diff
@@ -25,6 +25,43 @@ type BitbucketCloudClient struct {
 	logger  Log
 }
 
+// GetCommitStatus for a specific branch, NOTE: currently does not support pagination !
```

Comment:
Not only a branch but also a tag, commit hash, etc...
The full documentation is in the vcsclient.go.
```suggestion
// GetCommitStatus on Bitbucket cloud
```

About the pagination comment - it looks like it is not supported also on GitHub, GitLab, and Bitbucket cloud (maybe in Azure too). Can we add this comment in the vcsclient.go and in the README?